project/Dependencies.scala

```diff
@@ -16,10 +16,10 @@ object Dependencies {
   val junitVersion = "4.13.1"
   val slf4jVersion = "1.7.30"
   // check agrona version when updating this
-  val aeronVersion = "1.30.0"
+  val aeronVersion = "1.31.1"
   // needs to be inline with the aeron version, check
   // https://github.com/real-logic/aeron/blob/1.x.y/build.gradle
-  val agronaVersion = "1.7.2"
+  val agronaVersion = "1.8.0"
```

Comment:
hm, this was already changed in 1.30.0, so we must have missed Agrona in that update